sdk/src/main/java/com/horizen/vrf/VRFKeyGenerator.java

```diff
@@ -0,0 +1,10 @@
+package com.horizen.vrf;
+
+import com.horizen.utils.Pair;
+
+public class VRFKeyGenerator {
+
+  public static Pair<VRFSecretKey, VRFPublicKey> generate(byte[] seed) {
+    return new Pair(new VRFSecretKey(new byte[32]) , new VRFPublicKey(new byte[32]));
```

Comment:
Maybe, we need to do a jni call here that will return 2 byte arrays one for public another for private key